africanus/model/spectral/spec_model.py

```diff
@@ -170,8 +171,8 @@ def impl(stokes, spi, ref_freq, frequency, base=0):
                         spec_model = estokes[s, p]
 
                         for si in range(0, nspi):
-                            term = espi[s, si, p] * freq_ratio**(si + 1)
-                            spec_model += term
+                            term = (freq_ratio + 1) ** espi[s, si, p]
```

Comment:
How about changing the definition of ``freq_ratio`` above to

```python
freq_ratio = frequency[f] / rf
```

This save some FLOPS here:

```suggestion
                            term = freq_ratio**espi[s, si, p]
```

Reply:
Done.